### ISSUE TITLE
Wrong implementation of GlowBlock#getDrops() and #breakNaturally()

### ISSUE THREAD

**Issue by Tonodus** [GlowstoneMC/Glowstone-Legacy#620]
In Glowstone's current implementation the methods mentioned above act like a player breaking the block with his hand.

For all blocks needing a special tool to mine...
- this results in wrong results for GlowBlock#getDrops() [without parameter]
- this results in no drops if GlowBlock#breakNaturally() is called.
-  => this causes to no block drops if the block is destroyed by an explosion.

Broken by an explosion...
- stone should drop cobblestone
- iron/gold ore should drop their items
- diamond/restone ore should drop diamonds/redstone
- leaves should drop their saplings, apples, etc.
- etc.

So basically an explosion 'mines' blocks with the needed tool; the mentioned methods should act the same.
